descriptors/ModuleDescriptor-template.json

```diff
@@ -243,7 +249,11 @@
           "pathPattern": "/orders/pieces",
           "permissionsRequired": ["orders.pieces.item.post"],
           "modulePermissions": [
-            "orders-storage.pieces.item.post"
+            "orders-storage.pieces.item.post",
+            "orders-storage.po-lines.collection.get",
```

Comment:
`orders-storage.po-lines.item.get` and `orders-storage.purchase-orders.item.get` instead of `orders-storage.po-lines.collection.get`

Reply:
done